docker-compose.yml

```
@@ -0,0 +1,38 @@
+
+services:
+  mysqldb:
+    image: mysql:latest
+    container_name: book_store_container
+    restart: unless-stopped
+    env_file: .env
+    ports:
+      - ${MYSQLDB_LOCAL_PORT}:${MYSQLDB_DOCKER_PORT}
+    environment:
+      MYSQL_USER: ${MYSQLDB_USER}
+      MYSQL_PASSWORD: ${MYSQLDB_PASSWORD}
+      MYSQL_ROOT_PASSWORD: ${MYSQLDB_ROOT_PASSWORD}
+      MYSQL_DATABASE: ${MYSQLDB_DATABASE}
+    healthcheck:
+      test: [ "CMD", "mysqladmin", "ping", "-h", "localhost", "-u", "root", "-p${MYSQLDB_ROOT_PASSWORD}" ]
+      interval: 10s
+      timeout: 10s
+      retries: 3
+
+  app:
+    depends_on:
+      mysqldb:
+          condition: service_healthy
+    restart: on-failure
+    image: book_store-service
+    build: .
+    env_file: .env
+    ports:
+      - ${SPRING_LOCAL_PORT}:${SPRING_DOCKER_PORT}
+      - ${DEBUG_PORT}:${DEBUG_PORT}
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url":"jdbc:mysql://mysqldb:3306/${MYSQLDB_DATABASE}?createDatabaseIfNotExist=true",
+        "spring.datasource.username":"${MYSQLDB_USER}",
+        "spring.datasource.password":"${MYSQLDB_PASSWORD}"
+      }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```

Debug port mismatch: you map `${DEBUG_PORT}`, but the JVM option uses a fixed `5005`. Unify it:
```suggestion
      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:${DEBUG_PORT}"
```